pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -548,6 +548,14 @@ type ComponentSpec struct {
 	// - SLOW_LOG_FILE
 	// +optional
 	Env []corev1.EnvVar `json:"env,omitempty"`
+
+	// Additional containers of the component.
+	// +optional
+	Containers []corev1.Container `json:"containers,omitempty"`
```

Comment:
I think `Containers` here would be misleading for user. What about `Sidecars`?  As kubernetes support `sidecar` lifecycle since 1.18, this would be more clear for user to understand.

Reply:
Umm, what about `additionalContainers`?

Reply:
additionalContainers is ok to me.